tests/integration/test_execution.py

```diff
@@ -137,6 +139,11 @@ def test_ssh_and_scp(juju: jubilant.Juju):
     passwd = pathlib.Path('passwd').read_text()
     assert 'redis:' in passwd
 
+    # Test a round trip
+    juju.scp(__file__, 'snappass-test/0:/tmp/foobar.py')
+    juju.scp('snappass-test/0:/tmp/foobar.py', 'foobar.py')
+    assert pathlib.Path('foobar.py').read_text() == pathlib.Path(__file__).read_text()
```

Comment:
Nice trick, but I don't think this really tests temp dir for snaps.
IIRC github actions checkout is done to home directory, which snapped juju can read.

To test required tempifictation, have your **local path** under `/tmp/`.